CONTRIBUTING.md

```diff
@@ -0,0 +1,30 @@
+# Contributing to the eduNEXT TVM
+
+# If you've got only the idea
+If you do not have the ability to develop the idea but you need that solution, you will be able to [open a new feature request](https://github.com/eduNEXT/tvm/issues/new/choose)), if the idea is adopted by the community, this can be continued the next process, read __If you want contribute with code__
```

Comment:
Small typo
```suggestion
If you do not have the ability to develop the idea but you need that solution, you will be able to [open a new feature request](https://github.com/eduNEXT/tvm/issues/new/choose), if the idea is adopted by the community, this can be continued the next process, read __If you want contribute with code__
```